readme.md

```diff
@@ -23,35 +23,35 @@ $ git clone git@github.com:ga-wdi-exercises/whenpresident.git
 > The starter and solution code are branches in the [`WhenPresident` repo](https://github.com/ga-wdi-exercises/whenpresident/).
 
 
-#### Starter (express-mongoose-starter)
+#### Starter Code
 
 Checkout to the proper branch...
 
 ```bash
 $ git checkout express-mongoose-starter
 ```
 
-> Optionally, create a new branch on which you will do your work...
+Optionally, create a new branch on which you will do your work...
 
 ```bash
 $ git checkout -b MyName-express-mongoose
 ```
 
-#### And While You're At It...
+#### While You're At It...
 
-Install the modules listed in `package.json` and get Mongo running.
+Install the modules listed in `package.json`...
 
-Install dependencies
-  ```bash
-  $ npm install
-  ```
+```bash
+$ npm install
+```
 
-Then, start the mongo server - in **another** tab/window
-  ```bash
-  $ mongod  # Do this one in a separate tab or window.
-  ```
+Then, start the Mongo server in a separate Terminal tab or window...
 
-Check out what the starter code looks like in the browser by running `$ nodemon` and then visiting `http://localhost:3001`.
+```bash
+$ mongod
+```
+
+Check out what the starter code looks like in the browser by running `$ nodemon` and then visiting `http://localhost:3001` in the browser.
 
 ## Express Review (25 minutes / 0:30)
 
```

Comment:
The `Note` section needs to be updated. The Mongoose lesson had both promises and callbacks